tests/spread/plugins/copy/snaps/copy-plugin-test/snap/snapcraft.yaml

```diff
@@ -7,6 +7,7 @@ description: |
   Make a new one.
 
 grade: devel
+base: core18
```

Comment:
copy plugin is only available when not using bases, we can probably remove these and others that are not bases related as we do test them for releases to legacy. That said, this is also a quick smoke test to make sure we do not break re-execution.